index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+</head>
+
+<body>
+    <nav>
+        <h3 class="logo">My Logo</h3>
+        <img class="onion" src="image/Union.png" alt="" />
+    </nav>
+
+
+    <div class="container">
+
+        <section class="backgroundimg">
+            <div class="menuSection menuSection1">
+                <h2 class="introduction">I’m Michael Glad to see you!</h2>
+                <p class="Whoiam">
+                    I’m a software developer! I can help you build a product , feature
+                    or website Look through some of my work and experience! If you like
+                    what you see and have a project you need coded, don’t hestiate to
+                    contact me.
+                </p>
+            </div>
+            <div class="menuSection">
+                <a href="#" target="_blank">LET'S CONNECT</a>
+                <ul class="socialMedia">
+                    <li>
+                        <img src="image/Twitter.png" alt="" />
+                    </li>
+                    <li>
+                        <img src="image/Linkedin.png" alt="" />
+                    </li>
+                    <li>
+                        <img src="image/Meduim.png" alt="" />
+                    </li>
+                    <li>
+                        <img src="image/Github.png" alt="" />
+                    </li>
+                    <li>
+                        <img src="image/angellist.png" alt="" />
+                    </li>
+                </ul>
```

Comment:
- [ ] You did very well using `ul` and `li`. 👏 👏 
   All you need to do is wrap the `img` icons with an `a` tag because of [HTML semantics](https://www.simplilearn.com/tutorials/html-tutorial/html-semantics#:~:text=HTML%20Semantic%20Elements,-Tags&text=The%20element%20defines%20a,the%20section%20of%20the%20document.) and apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

You got this 💪